storage-node/src/server.rs

```diff
@@ -1,26 +1,105 @@
 use enum_as_inner::EnumAsInner;
+use std::fs;
+use std::path::Path;
+use storage_common::RequestParams;
+use tokio::fs::File as AsyncFile;
+use tokio_util::io::ReaderStream;
+use warp::{Filter, Rejection, Reply};
 
 use crate::{
     cache::lru::LruCache, disk::disk_manager_sync::DiskManagerSync, error::ParpulseResult,
     storage_manager::StorageManager,
 };
 
-// FIXME: Discuss with catalog team for more information.
-#[derive(Clone, EnumAsInner)]
-pub enum RequestParams {
-    File(String),
-    S3(String),
-}
-
 pub async fn storage_node_serve() -> ParpulseResult<()> {
     // TODO: Read the type of the cache from config.
     let dummy_size = 10;
     let cache = LruCache::new(dummy_size);
     let disk_manager = DiskManagerSync::default();
     let storage_manager = StorageManager::new(cache, disk_manager, "cache/".to_string());
 
-    // TODO:
-    // 1. Start the server here and listen on the requests.
-    // 2. Feed the request into the storage manager.
-    todo!()
+    // FIXME (kunle): We need to get the file from storage manager. For now we directly read
+    // the file from disk. I will update it after the storage manager provides the relevant API.
+    let route = warp::path!("file" / String)
+        .and(warp::path::end())
+        .and_then(|file_name: String| async move {
+            // let file_path = format!("storage-node/tests/parquet/{}", file_name);
```

Comment:
```suggestion
```

Reply:
I need to fix this in the e2e test. The working directory will be different between running e2e and unit tests. So cannot hard code it.